src/main/groovy/liquibase/harness/generateChangelog/GenerateChangelogTestHelper.groovy

```diff
@@ -110,9 +104,6 @@ class GenerateChangelogTestHelper {
         String password
         String url
         String setupChangelogPath
```

Comment:
`setupChangelogPath` and `dbSchema` are also part of stress test, not needed in generateChangelog

Reply:
done